Source/SpinnerView.swift

```diff
@@ -87,14 +87,20 @@ public class SpinnerView: UIView {
     }
     
     func startAnimating() {
-        activityIndicator?.removeFromSuperview()
-        activityIndicator = MaterialActivityIndicatorView()
-        if let view = activityIndicator {
-            addSubview(view)
+        DispatchQueue.main.async { [weak self] in
```

Comment:
spinner animation on swipe to move forward not working. after adding a delay of `0.1`, it is working.

```suggestion
        DispatchQueue.main.asyncAfter(deadline: .now() { [weak self] in
```

Reply:
Its working best with a delay of `0.4` seconds.